showcase/src/assets/translations/en.json

```diff
@@ -98,10 +98,15 @@
           "INPUT_ONLY_NUMBERS": "Accept only numbers",
           "INPUT_ALPHANUMERICAL_CHARACTERS": "Accept only alphanumeric characters",
           "INPUT_NO_SPECIAL_CHARACTERS": "Accept all except special characters",
-          "INPUT_UPPERCASE_CHARACTERS": "Accept only uppercase characters",
-          "TYPE_A_VALUE": "Type a value"
+          "INPUT_UPPERCASE_CHARACTERS": "Accept only uppercase characters"
         },
-        "TITLE": "Keyboard directive"
+        "TRANSFORM_INPUT": {
+          "TITLE": "Transform Input Directive",
+          "ONLY_UPPER_CASE_LABEL": "Upper case only",
```

Comment:
Maybe putting "Value transformed to uppercase" will be more descriptive because I would think that "Upper case only" means that I cannot type lowercase case chars but I do it is just that they are transformed to uppercase... what do you think?

Also you can add in this label to the label "(FormControl)" to make it clear that a FormControl is used for that field